src/js/tabs/motors.js

```diff
@@ -558,8 +558,12 @@ TABS.motors.initialize = function (callback) {
 
             $('div.values li').eq(index).text($(this).val());
 
-            for (var i = 0; i < 8; i++) {
+            for (var i = 0; i < FC.MOTOR_CONFIG.motor_count; i++) {
                 var val = parseInt($('div.sliders input').eq(i).val());
+                // adjust val to retain the same while switching tabs
+                if (val !== 1000){
+                    val++;
```

Comment:
While this obviously is a fix to how the problem manifests (aka. the symptom), it would be nice to know why this is the case - potentially there are more underlying problems.

Did you investigate and find out what causes this weird behaviour?

Reply:
Still didn't find the cause. But beside switching between tabs it also fixes the following behavior when using sliders:

Slider | Bar
--- | ---
1000 | 1000
1001 | 1001
1002 | 1001
1003 | 1002
1004 | 1003
1998 | 1997
1999 | 1998
2000 | 2000

Only 1999 and 2000 will now give 2000 but I guess this edge won't be as important as the start of the slider.